modules/loader/source/relocations.cpp

```diff
@@ -80,8 +85,8 @@ inline Integer setBitRange(Integer value, Integer subvalue, int first,
 
 // Returns a triple of common variables used in relocation calculation:
 // * Writable position in for relocations (host memory)
-// * P - position of the relocation (target memory)
-// * S - value of the symbol in the symbol table (target memory)
+// * P:    : position of the relocation (target memory)
+// * S + A : value of the symbol in the symbol table (target memory)
```

Comment:
why is there two values S and A, what's the difference?

Reply:
I get you are using known elf symbols, but I don't think it does any harm to say what these values are

Reply:
I've expanded the comment to say it returns `S + A`, "the value of the symbol in the symbol table plus any relocation addend" which I think covers it